libs/logic-apps-shared/src/utils/src/lib/helpers/functions.ts

```diff
@@ -1037,6 +1037,25 @@ export const filterRecord = <T>(data: Record<string, T>, filter: (_key: string,
     }, {});
 };
 
+/**
+ * Returns a sorted record object.
+ * @param data - The record to sort.
+ * @param sort - The sort function.
+ * @returns {Record<string, T>} - The sorted record.
+ */
+
+export const sortRecord = <T>(
```

Comment:
NIT: should we start adding tests whenever we add reusable functions in this file?